### PR TITLE
Fix: Logo on small screen in overlayHeader should only show fast flag

### PR DESCRIPTION
**Pull request by johannacarlberg**
## 🖼 Context

https://transferwise.atlassian.net/browse/AT-2916

## ✅ Checklist

- [x] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [x] Changes are tested and all tests pass
- [x] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [x] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/components/src/logo/Logo.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
+import { useWindowSize } from 'react-use';
```
Comment (dozortsev) [inline packages/components/src/logo/Logo.js:3]: we have own equivalent of this, called `useClientWidth`

```ts
const [clientWidth] = useClientWidth({ ref: isServerSideRendering() ? undefined : window })
```

File: packages/components/src/logo/Logo.js [hunk 2]
```diff
@@ -10,10 +13,13 @@ const logoPaths = {
   WISE_BUSINESS: 'brand_logo_business.svg',
   WISE_INVERSE: 'brand_logo_inverse.svg',
   WISE_BUSINESS_INVERSE: 'brand_logo_business_inverse.svg',
+  WISE_FLAG: 'brand_flag.svg',
```
Comment (dozortsev) [inline packages/components/src/logo/Logo.js:16]: great,

but now we have duplicate logic in FlowNavigation which we need to delete from there and reuse logic of Logo component

I'm talking about these styles https://github.com/transferwise/neptune-web/blob/main/packages/components/src/flowNavigation/FlowNavigation.js#L46-L50 they load `brand_flag.svg` via CSS https://github.com/transferwise/neptune-web/blob/main/packages/components/src/flowNavigation/FlowNavigation.less#L74-L94

File: packages/components/src/logo/Logo.js (hunk 1 above)
Comment (johannacarlberg) [reply packages/components/src/logo/Logo.js:3]: ah nice will use that instead 👍

**Review by dozortsev — COMMENTED** [review]
thanks for contributing, it looks great 🙇

File: packages/components/src/logo/Logo.js (hunk 2 above)
Comment (johannacarlberg) [reply packages/components/src/logo/Logo.js:16]: 👍  will address

File: packages/components/src/flowNavigation/FlowNavigation.js [hunk 3]
```diff
@@ -43,12 +43,7 @@ const FlowNavigation = ({ activeStep, avatar, logo, onClose, onGoBack, done, ste
             onClick={onGoBack}
           />
         )}
-        <div
-          className={classNames('np-flow-navigation--flag', {
-            'np-flow-navigation--flag__hidden': displayGoBack,
-            'np-flow-navigation--flag__display': !displayGoBack,
-          })}
-        />
+        {!displayGoBack && <>{logo}</>}
```
Comment (dozortsev) [inline packages/components/src/flowNavigation/FlowNavigation.js:46]: could use ternary operator here

```ts
{displayGoBack ? (<BackButton...) : logo}
```

**Review by dozortsev — APPROVED** [review]
thanks a lot!

**Comment by johannacarlberg** [comment]
thanks @dozortsev , changed it to a ternary!

**Pull request merged**